src/smc-hub/share/mathjax-support.coffee

```diff
@@ -8,8 +8,24 @@ async   = require('async')
 console.log("loading mathjax-node...")
 mathjax = require('mathjax-node')
 
-# TODO: see
-# mathjax.config(...)
+# Deriving MyMathJaxConfig from the webapp's MathJaxConfig
+# It's the same, execpt for the entries that reference additional .js files
+# Otherwise the rendered page wouldn't even show up, so I assume there is no support for that
```

Comment:
This all happens on the backend in a jsdom that mathjax-node creates.  It's probably possible to have it load extensions, but I don't know.